### DISCUSSION TITLE
Issue with the link at https://pmd.github.io/pmd-6.33.0/pmd_rules_java.html

### DISCUSSION THREAD

**Discussion by anitsh** [pmd/pmd#3223] [General]
Well I was not sure if it would be right to raise as an issue.

### Issue
![image](https://user-images.githubusercontent.com/414141/115106536-1a8f5580-9f85-11eb-86a8-6f1eff847d81.png)

Page opened from "Edit me" does not exists.

**Comment by oowekyala** [comment]
Thanks for the feedback! Since this page is generated, there's no source to edit. So we somehow should remove this button. I think we should reopen this as an issue

**Reply by adangel** [reply]
Yes, the overview page like https://pmd.github.io/latest/pmd_rules_java.html is completely generated and there we should remove the button.

The individual category pages like https://pmd.github.io/latest/pmd_rules_java_bestpractices.html contain a "Edit me" button, which works. It opens the category ruleset xml file.

**Reply by adangel** [reply]
Created #3230
